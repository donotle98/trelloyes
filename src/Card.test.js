import React from "react";
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";
import Card from "./Card";
import store from "./store";

describe("here is the test for cards", () => {
  it("renders without crashing", () => {
    const div = document.createElement("div");

    ReactDOM.render(<Card />, div);

    ReactDOM.unmountComponentAtNode(div);
  });
  it("renders the UI as expected", () => {
    const tree = renderer
      .create(
        <Card title={store.allCards.title} content={store.allCards.content} />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
