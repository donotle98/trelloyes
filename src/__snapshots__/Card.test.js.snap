// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`here is the test for cards renders the UI as expected 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3 />
  <p />
</div>
`;
